cv_next/server/api/comments.ts

```diff
@@ -1,19 +1,11 @@
-import 'server-only'
+import "server-only"
```

Comment:
Note that the changes in this file are probably just mis-configured prettier changes

Reply:
The entire class still uses Firebase